code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react'
+import { useParams, Link } from 'react-router-dom'
+import backicon from '../icons/backicon.png'
+
+export const MovieDetails = () => {
+  const [movieDetails, setMovieDetails] = useState()
+  const { id } = useParams()
+  
+
+  useEffect(() => {
+    fetch (`https://api.themoviedb.org/3/movie/${id}?api_key=e87ff9a70deb7f8f6f897b1efba6340a&language=en-US`)
+    .then ((res) => res.json())
+    .then ((json) => {setMovieDetails(json)}) 
+    .catch(err => console.error(err))
+  }, [id])
+
+
+  if(movieDetails === undefined) {
+    return   <></>
+ }
+
+  return (
+
```

Comment:
It might be a bit of a personal taste thing, but ESLint isn't to happy about blank rows.